inspirehep/config.py

```diff
@@ -760,7 +764,8 @@ def _(x):
             view="invenio_oauthclient.handlers:signup_handler",
         ),
         params=dict(
-            request_token_params={'scope': '/authenticate'},
+            request_token_params={
+                'scope': '/orcid-profile/read-limited /activities/update /orcid-bio/update'},
```

Comment:
Because it's a subset of the other scopes... See bottom of https://members.orcid.org/api/orcid-scopes